Tools/Frame_params/hexsoon-td650.param

```diff
@@ -0,0 +1,29 @@
+#NOTE: Hexsoon TD650 params for ArduPilot Copter 4.1
+ANGLE_MAX,3500
+ACRO_YAW_P,2.6
+ATC_ACCEL_P_MAX,86000
+ATC_ACCEL_R_MAX,86000
+ATC_ACCEL_Y_MAX,23400
+ATC_RAT_PIT_FLTD,17

```

Comment:
We should probably also increase the ATC_ANG_RLL_P and ATC_ANG_PIT_P values from 4.5 to at least 6.0 I think.